source/reference/api/reverse.txt

```diff
@@ -99,3 +104,25 @@ entire sync process to copy the original data.
 To view the mapping direction for the synchronization of the source and
 destination clusters, use the :ref:`progress <c2c-api-progress>`
 endpoint and check the ``directionMapping`` object.
+
+.. _c2c-validate-unique-index:
+
+Validate Unique Indexes
```

Comment:
[question]

I'm curious why we're separating this content from the Unique Indexes bullet in the [Requirements](https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-32087-reverse-req/reference/api/reverse/#requirements) section. They seem to repeat some of the same information, and the details in this section seem like requirements, or something I'd need to do before I run the reverse endpoint.

Rather than having this information in two places, would it be reasonble to move all of this info to the Requirements section?

Reply:
Fixed